.ci/butler.groovy

```diff
@@ -115,31 +115,35 @@ def call(config) {
 	ret = sh(returnStatus: true,
 		script:"""#!/bin/bash
 							cd ${config.repository_root}/.ci
-							./gitler.sh""")
+							./butler.sh""")
 
 	if (fileExists(config.repository_root+'restyling.patch')) {
 		emailext (
-			subject: "Job '${env.JOB_NAME} #${env.BUILD_NUMBER} [PR#${env.CHANGE_ID}]' failed due to bad code styling",
-			body: """<p>Job '${env.JOB_NAME} [<a href="${env.CHANGE_URL}">PR#${env.CHANGE_ID}</a> - ${env.CHANGE_TITLE}]' failed because code style does not follow the standards.</p>
-			A patch to rectify the errors is attached. You apply the patch using:<br>
+			subject: "PR#${env.CHANGE_ID} - ${env.CHANGE_TITLE} has unfortunate code styling",
+			body: """<p>Greetings Sir!<p>
```

Comment:
This assumes the person submitting the PR is a man. Shouldn't we try to be as inclusive as possible?

Reply:
Would it hurt the text to remove the use of "Sir" by the butler?

Reply:
The butler uses Master in "Well done Master" which isn't very gender-neutral either. Would be nice to use the same word everywhere.
Maybe use "Highness" or "Your Grace"?

We want the butler to be perceived as helpful and polite (instead of harsh and demanding) so some sort of polite expression would be nice.

Reply:
Well, Well done Master is exclusive for core team members, and last time I checked, it was not very gender-neutral either.
If Sir is not good enough, please suggest alternatives. "Highness" and "Your Grace" sounds a bit odd and overly arcane I think.

Reply:
I would remove all titles altogether.

Reply:
Ok, lets go for that. I see the PR is already merged. Will you file a new one?

Reply:
Ok, I'll take a shot.